awx/ui/client/lib/models/UnifiedJobTemplate.js

```diff
@@ -0,0 +1,110 @@
+let BaseModel;
```

Comment:
TODO: this will be superseded by @mabashian's work...need to figure out best way to deal with that in the context of this PR.